### ISSUE TITLE
Incorrectly identify queryset as not iterable when using exclude.

### ISSUE THREAD

**Issue by simonpanay** [PyCQA/pylint-django#177]
[pip_freeze.txt](https://github.com/PyCQA/pylint-django/files/2189146/pip_freeze.txt)

```python
return [periodique.pk for periodique in Periodique.objects.exclude(periode__isnull=True)]
```
with this code, pylint detects the following error : 
pylint: not-an-iterable / Non-iterable value Periodique.objects.exclude(periode__isnull=True) is used in an iterating context (col 48)

If I replace exclude by filter, no error is detected by pylint

**Comment by atodorov** [comment]
Can you retry with pylint-django 2.0 that was released today ?
